server/api/constants.go

```diff
@@ -7,11 +7,14 @@ package api
 const (
 	Repository     = "mattermost-plugin-apps"
 	CommandTrigger = "apps"
+	ManifestsFile  = "manifests.json"
```

Comment:
Is it replacing the `apps.json`? (in which case we need to update the references)